stglib/tests/test_RBR_VTu.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+import os
+# the following three lines are necessary to run in an environment
+# where stglib is not on the path
+import sys
+filepath = os.path.join(os.path.dirname(os.path.abspath(__file__)), 
+                        '..\\..\\..\\stglib')
+sys.path.append(filepath)
+# xml to dict is a required module in order to import stglib.  
+# As it is not a part of Anaconda's IOOS distribution, 
+# nor is it likely to be, you may need this
```

Comment:
You won't (shouldn't) need this if stglib is installed in the path, which it should be. Please remove these hard-coded lines especially for the tests and they should work. Also please do not use double-escaped backslashes to specfy path, use forward slashes (/) instead, which should still work fine on a windows python install.